content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node can run the Pod and multiple scheduling retries may be
+needed.
+
+<!-- body -->
+
+
+## Enabling the feature
```

Comment:
I would move this to the foot of the page, but add text to the introduction that links to that section.

Also, I'd title this:
```suggestion
## Enabling storage capacity tracking
```

Reply:
I think the section is important enough for users that it should comes first. The key point is that it's an optional feature and something that needs to be part of the CSI driver deployment. Once users know that, they can decide whether want to read the rest of the page or first consult their CSI driver.